roles/network_plugin/fan/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: "Fan | Ensure ubuntu-fan installed"
+  apt:
+    name: "ubuntu-fan"
```

Comment:
Yeah, i can see why that would be problematic. There's other stuff as well, which makes this a very opinionated overlay network, and i'm not interested in getting it packaged for other distros myself, so probably this should be kept out of the main repository -- not sure if it could be packaged "better" as a plugin, because it needs to fiddle with files for kubelet as well, so there's no clear way of getting this installed by "dropping it in" as an external plugin kinda thing ..